sdk/nodejs/appservice/zMixins.ts

```diff
@@ -711,3 +712,57 @@ export function getResourceGroupNameAndLocation(
     const getResult = resourceGroupName.apply(n => core.getResourceGroup({ name: n }));
     return { resourceGroupName, location: getResult.location };
 }
+
+/**
+ * Keys associated with a Function App.
+ */
+export interface FunctionHostKeys {
+    /** Master key. */
+    masterKey: string;
+    /** A dictionary of system keys, e.g. for Durable Functions or Event Grid. */
+    systemKeys: { [key: string]: string };
+    /** Default function keys. */
+    functionKeys: FunctionKeys;
+}
+
+/**
+ * Keys associated with a single Function.
+ */
+export interface FunctionKeys {
+    default: string;
+    [key: string]: string;
+}
+
+declare module "./functionApp" {
+    interface FunctionApp {
+        /**
+         * Retrieve the keys associated with the Function App.
+         */
+        listHostKeys(): pulumi.Output<FunctionHostKeys>;

```

Comment:
'list' seems odd.  why not just 'get'?

Reply:
That's the term they use, might make it a bit better discoverable? Otherwise, agree.

Reply:
Renamed

Reply:
ok.  i'm ok either way.  i leave it to you what you feel is best.  i personally think 'get' is totally fine.